redis_cloud/metadata.csv

```diff
@@ -0,0 +1,139 @@
+rdsc.listener_write_req_total,rate,15,request,,"Accumulative latency (sum of the latencies) of commands that are type “read” on DB. For the average latency, divide this value by rdsc.listener_read_res",1,redis_cloud_listener,listener_write_req_total,
+rdsc.listener_write_req_max_total,gauge,15,request,,Highest value of accumulative latency of commands that are type “read” on DB,1,redis_cloud_listener,listener_write_req_max_total,
+rdsc.listener_write_res_total,rate,15,response,,"Accumulative latency (sum of the latencies) of commands that are type “write” on DB. For the average latency, divide this value by rdsc.listener_write_res",1,redis_cloud_listener,listener_write_res_total,
+rdsc.listener_write_res_max_total,gauge,15,response,,Highest value of accumulative latency of commands that are type “write” on DB,1,redis_cloud_listener,listener_write_res_max_total,
+rdsc.listener_write_started_res_total,rate,15,response,,Number of memcached AUTH commands sent to the DB,1,redis_cloud_listener,listener_write_started_res_total,
+rdsc.listener_write_started_res_max_total,gauge,15,response,,Highest value of number of memcached AUTH commands sent to the DB,1,redis_cloud_listener,listener_write_started_res_max_total,
+rdsc.dmcproxy_process_cpu_system_seconds_total,count,15,second,,Total CPU time used by the proxy,1,redis_cloud_operations,dmcproxy_process_cpu_system_seconds_total,
+rdsc.dmcproxy_process_cpu_usage_percent,gauge,15,percent,,Percent CPU used by the proxy,0,redis_cloud_operations,dmcproxy_process_cpu_usage_percent,
+rdsc.dmcproxy_process_cpu_user_seconds_total,count,15,second,,Total seconds the proxy used CPU,1,redis_cloud_operations,dmcproxy_process_cpu_user_seconds_total,
+rdsc.dmcproxy_process_main_thread_cpu_system_seconds_total,count,15,second,,Total seconds the proxy main thread used CPU,1,redis_cloud_operations,dmcproxy_process_main_thread_cpu_system_seconds_total,
+rdsc.dmcproxy_process_main_thread_cpu_user_seconds_total,count,15,second,,Total seconds the proxy main thread used CPU,1,redis_cloud_operations,dmcproxy_process_main_thread_cpu_user_seconds_total,
+rdsc.dmcproxy_process_max_fds,count,15,file,,Maximum number of file descriptors used by the proxy,1,redis_cloud_operations,dmcproxy_process_max_fds,
+rdsc.dmcproxy_process_open_fds,gauge,15,file,,Maximum number of file descriptors used by the proxy,0,redis_cloud_operations,dmcproxy_process_open_fds,
+rdsc.dmcproxy_process_resident_memory_bytes,gauge,15,byte,,Resident memory used by the proxy,0,redis_cloud_operations,dmcproxy_process_resident_memory_bytes,
+rdsc.dmcproxy_process_start_time_seconds,gauge,15,second,,Number of seconds since the proxy started,0,redis_cloud_operations,dmcproxy_process_start_time_seconds,
+rdsc.dmcproxy_process_virtual_memory_bytes,gauge,15,byte,,Virtual memory used by the proxy,0,redis_cloud_operations,dmcproxy_process_virtual_memory_bytes,
```

Comment:
```suggestion
rdsc.bdb_conns,count,15,connection,,Number of client connections to the DB,0,redis_cloud_database,bdb_conns,
rdsc.bdb_egress_bytes,rate,15,byte,,Rate of outgoing network traffic from the DB (bytes/sec),0,redis_cloud_database,bdb_egress_bytes,
rdsc.bdb_egress_bytes_max,gauge,15,byte,,Highest value of rate of outgoing network traffic from the DB (bytes/sec),1,redis_cloud_database,bdb_egress_bytes_max,
rdsc.bdb_evicted_objects,rate,15,eviction,,Rate of key evictions from the DB (evictions/sec),0,redis_cloud_database,bdb_evicted_objects,
rdsc.bdb_evicted_objects_max,gauge,15,eviction,,Highest value of rate of key evictions from the DB (evictions/sec),1,redis_cloud_database,bdb_evicted_objects_max,
rdsc.bdb_expired_objects,rate,15,eviction,,Rate of keys expired in the DB (expirations/sec),0,redis_cloud_database,bdb_expired_objects,
rdsc.bdb_expired_objects_max,gauge,15,eviction,,Highest value of rate of keys expired in the DB (expirations/sec),1,redis_cloud_database,bdb_expired_objects_max,
rdsc.bdb_fork_cpu_system,rate,15,cpu,,% cores utilization in system mode for all Redis shard fork child processes of this database,0,redis_cloud_database,bdb_fork_cpu_system,
rdsc.bdb_fork_cpu_system_max,gauge,15,cpu,,Highest value of % cores utilization in system mode for all Redis shard fork child processes of this database,1,redis_cloud_database,bdb_fork_cpu_system_max,
rdsc.bdb_fork_cpu_user,rate,15,cpu,,% cores utilization in user mode for all Redis shard fork child processes of this database,0,redis_cloud_database,bdb_fork_cpu_user,
rdsc.bdb_fork_cpu_user_max,gauge,15,cpu,,Highest value of % cores utilization in user mode for all Redis shard fork child processes of this database,1,redis_cloud_database,bdb_fork_cpu_user_max,
rdsc.bdb_ingress_bytes,count,15,byte,,Rate of incoming network traffic to the DB (bytes/sec),0,redis_cloud_database,bdb_ingress_bytes,
rdsc.bdb_ingress_bytes_max,gauge,15,byte,,Highest value of rate of incoming network traffic to the DB (bytes/sec),1,redis_cloud_database,bdb_ingress_bytes_max,
rdsc.bdb_instantaneous_ops_per_sec,count,15,operation,,Request rate handled by all shards of the DB (ops/sec),0,redis_cloud_database,bdb_instantaneous_ops_per_sec,
rdsc.bdb_main_thread_cpu_system,rate,15,cpu,,% cores utilization in system mode for all Redis shard main threads of this database,0,redis_cloud_database,bdb_main_thread_cpu_system,
rdsc.bdb_main_thread_cpu_system_max,gauge,15,cpu,,Highest value of % cores utilization in system mode for all Redis shard main threads of this database,1,redis_cloud_database,bdb_main_thread_cpu_system_max,
rdsc.bdb_main_thread_cpu_user,rate,15,cpu,,% cores utilization in user mode for all Redis shard main threads of this database,0,redis_cloud_database,bdb_main_thread_cpu_user,
rdsc.bdb_main_thread_cpu_user_max,gauge,15,cpu,,Highest value of % cores utilization in user mode for all Redis shard main threads of this database,1,redis_cloud_database,bdb_main_thread_cpu_user_max,
rdsc.bdb_mem_frag_ratio,rate,15,percent,,RAM fragmentation ratio (RSS / allocated RAM),0,redis_cloud_database,bdb_mem_frag_ratio,
rdsc.bdb_mem_not_counted_for_evict,count,15,byte,,Resident Memory,0,redis_cloud_database,bdb_mem_not_counted_for_evict,
rdsc.bdb_mem_size_lua,gauge,15,byte,,Redis Lua scripting heap size (bytes),0,redis_cloud_database,bdb_mem_size_lua,
rdsc.bdb_memory_limit,count,15,byte,,Configured RAM limit for the database (bytes),0,redis_cloud_database,bdb_memory_limit,
rdsc.bdb_monitor_sessions_count,count,15,key,,Number of clients connected in monitor mode to the DB,1,redis_cloud_database,bdb_monitor_sessions_count,
rdsc.bdb_no_of_expires,count,15,key,,Current number of volatile keys in the database,0,redis_cloud_database,bdb_no_of_expires,
rdsc.bdb_no_of_keys,count,15,key,,Number of keys in the DB,0,redis_cloud_database,bdb_no_of_keys,
rdsc.bdb_other_req,rate,15,operation,,Rate of other (non-read/write) requests on the DB (ops/sec),0,redis_cloud_database,bdb_other_req,
rdsc.bdb_other_req_max,gauge,15,operation,,Highest value of rate of other (non-read/write) requests on the DB (ops/sec),1,redis_cloud_database,bdb_other_req_max,
rdsc.bdb_other_res,rate,15,operation,,Rate of other (non-read/write) responses on the DB (ops/sec),0,redis_cloud_database,bdb_other_res,
rdsc.bdb_other_res_max,gauge,15,operation,,Highest value of rate of other (non-read/write) responses on the DB (ops/sec),1,redis_cloud_database,bdb_other_res_max,
rdsc.bdb_pubsub_channels,count,15,key,,Count of the pub/sub channels with subscribed clients,0,redis_cloud_database,bdb_pubsub_channels,
rdsc.bdb_pubsub_channels_max,gauge,15,key,,Highest value of count of the pub/sub channels with subscribed clients,1,redis_cloud_database,bdb_pubsub_channels_max,
rdsc.bdb_pubsub_patterns,count,15,key,,Count of the pub/sub patterns with subscribed clients,0,redis_cloud_database,bdb_pubsub_patterns,
rdsc.bdb_pubsub_patterns_max,gauge,15,key,,Highest value of count of the pub/sub patterns with subscribed clients,1,redis_cloud_database,bdb_pubsub_patterns_max,
rdsc.bdb_read_hits,count,15,hit,,Rate of read operations accessing an existing key (ops/sec),0,redis_cloud_database,bdb_read_hits,
rdsc.bdb_read_hits_max,gauge,15,hit,,Highest value of rate of read operations accessing an existing key (ops/sec),1,redis_cloud_database,bdb_read_hits_max,
rdsc.bdb_read_misses,count,15,miss,,Rate of read operations accessing a non-existing key (ops/sec),0,redis_cloud_database,bdb_read_misses,
rdsc.bdb_read_misses_max,gauge,15,miss,,Highest value of rate of read operations accessing a non-existing key (ops/sec),1,redis_cloud_database,bdb_read_misses_max,
rdsc.bdb_read_req,rate,15,get,,Rate of read requests on the DB (ops/sec),0,redis_cloud_database,bdb_read_req,
rdsc.bdb_read_req_max,gauge,15,get,,Highest value of rate of read requests on the DB (ops/sec),1,redis_cloud_database,bdb_read_req_max,
rdsc.bdb_read_res,rate,15,get,,Rate of read responses on the DB (ops/sec),0,redis_cloud_database,bdb_read_res,
rdsc.bdb_read_res_max,gauge,15,get,,Highest value of rate of read responses on the DB (ops/sec),1,redis_cloud_database,bdb_read_res_max,
rdsc.bdb_shard_cpu_system,rate,15,cpu,,% cores utilization in system mode for all Redis shard processes of this database,0,redis_cloud_database,bdb_shard_cpu_system,
rdsc.bdb_shard_cpu_system_max,gauge,15,cpu,,Highest value of % cores utilization in system mode for all Redis shard processes of this database,1,redis_cloud_database,bdb_shard_cpu_system_max,
rdsc.bdb_shard_cpu_user,rate,15,cpu,,% cores utilization in user mode for the Redis shard process,0,redis_cloud_database,bdb_shard_cpu_user,
rdsc.bdb_shard_cpu_user_max,gauge,15,cpu,,Highest value of % cores utilization in user mode for the Redis shard process,1,redis_cloud_database,bdb_shard_cpu_user_max,
rdsc.bdb_shards_used,rate,15,resource,,Number of shards in use,0,redis_cloud_database,bdb_shards_used,
rdsc.bdb_total_connections_received,count,15,connection,,Rate of new client connections to the DB (connections/sec),0,redis_cloud_database,bdb_total_connections_received,
rdsc.bdb_total_connections_received_max,gauge,15,connection,,Highest value of rate of new client connections to the DB (connections/sec),1,redis_cloud_database,bdb_total_connections_received_max,
rdsc.bdb_total_req,count,15,key,,Rate of all requests on the DB (ops/sec),0,redis_cloud_database,bdb_total_req,
rdsc.bdb_total_req_max,gauge,15,key,,Highest value of rate of all requests on the DB (ops/sec),1,redis_cloud_database,bdb_total_req_max,
rdsc.bdb_total_res,count,15,key,,Rate of all responses on the DB (ops/sec),0,redis_cloud_database,bdb_total_res,
rdsc.bdb_total_res_max,gauge,15,key,,Highest value of rate of all responses on the DB (ops/sec),1,redis_cloud_database,bdb_total_res_max,
rdsc.bdb_uptime_in_seconds,rate,15,second,,Uptime duration of the database (seconds),0,redis_cloud_database,bdb_uptime_in_seconds,
rdsc.bdb_write_req,rate,15,set,,Rate of write requests on the DB (ops/sec),0,redis_cloud_database,bdb_write_req,
rdsc.bdb_write_req_max,gauge,15,set,,Highest value of rate of write requests on the DB (ops/sec),1,redis_cloud_database,bdb_write_req_max,
rdsc.bdb_write_res,rate,15,set,,Rate of write responses on the DB (ops/sec),0,redis_cloud_database,bdb_write_res,
rdsc.bdb_write_res_max,gauge,15,set,,Highest value of rate of write responses on the DB (ops/sec),1,redis_cloud_database,bdb_write_res_max,
rdsc.mcs_avg_cache_item_size,gauge,60,byte,,The average item size stored in the cache,0,memcached,avg_cache_item_size,
rdsc.mcs_cache_hit_ratio,gauge,60,percent,,The ratio of cache hits to cache lookups,0,memcached,cache_hit_ratio,
rdsc.mcs_cache_hits,rate,60,operation,,Number of successful cache lookups,0,memcached,cache_hits,
rdsc.mcs_cache_hits_max,gauge,60,operation,,Highest value of cache hits in a time period,1,memcached,cache_hits_max,
rdsc.mcs_cache_items,count,60,item,,Number of items currently stored in the cache,0,memcached,cache_items,
rdsc.mcs_cache_misses,rate,60,operation,,Number of unsuccessful cache lookups,0,memcached,cache_misses,
rdsc.mcs_cache_misses_max,gauge,60,operation,,Highest value of cache misses in a time period,1,memcached,cache_misses_max,
rdsc.mcs_cache_req,rate,60,operation,,Rate of requests to the cache,0,memcached,cache_req,
rdsc.mcs_cache_res,rate,60,operation,,Rate of responses from the cache,0,memcached,cache_res,
rdsc.mcs_cache_size,gauge,60,byte,,Total size of the cache,0,memcached,cache_size,
rdsc.mcs_current_conn,gauge,60,connection,,Number of current connections to the cache,0,memcached,current_conn,
rdsc.mcs_egress_bytes,rate,60,byte,,Rate of outgoing network traffic from the cache (bytes/sec),0,memcached,egress_bytes,
rdsc.mcs_expired_objects,rate,60,expiration,,Rate of expired objects in the cache (expirations/sec),0,memcached,expired_objects,
rdsc.mcs_evicted_objects,rate,60,eviction,,Rate of evicted objects from the cache (evictions/sec),0,memcached,evicted_objects,
rdsc.mcs_hitrate,gauge,60,percent,,Ratio of cache hits to the sum of hits and misses,0,memcached,hitrate,
rdsc.mcs_ingress_bytes,rate,60,byte,,Rate of incoming network traffic to the cache (bytes/sec),0,memcached,ingress_bytes,
rdsc.mcs_mem_frag_ratio,rate,60,percent,,RAM fragmentation ratio (RSS / allocated RAM) in cache,0,memcached,mem_frag_ratio,
rdsc.mcs_memory_used,gauge,60,byte,,Total memory used by the cache,0,memcached,memory_used,
rdsc.mcs_read_req,rate,60,get,,Rate of read requests to the cache,0,memcached,read_req,
rdsc.mcs_read_res,rate,60,get,,Rate of read responses from the cache,0,memcached,read_res,
rdsc.mcs_total_connections_received,rate,60,connection,,Rate of new client connections to the cache (connections/sec),0,memcached,total_connections_received,
rdsc.mcs_uptime_in_seconds,rate,60,second,,Uptime duration of the cache (seconds),0,memcached,uptime_in_seconds,
rdsc.mcs_write_req,rate,60,set,,Rate of write requests to the cache,0,memcached,write_req,
rdsc.mcs_write_res,rate,60,set,,Rate of write responses from the cache,0,memcached,write_res,
```

Reply:
could you explain very briefly what the difference is?